tests_integration/runPrettier.js

```diff
@@ -50,12 +57,14 @@ function runPrettier(dir, args, options) {
   process.argv = ["path/to/node", "path/to/prettier/bin"].concat(args);
 
   jest.resetModules();
-  jest.setMock("get-stream", () => ({
-    then: handler => handler(options.input || "")
-  }));
+  jest
+    .spyOn(require(prettierApi).__debug, "getStream")
+    .mockImplementation(() => ({
+      then: handler => handler(options.input || "")
+    }));
```

Comment:
@ikatyang All the other `.spyOn(...).mockImplementation(...)` are _above_ the `jest.resetModules()` call. Does this matter in any way?

Reply:
The `jest.resetModules()` is for `require`s, it should not matter.